src/components/Calc.js

```diff
@@ -0,0 +1,139 @@
+              <button type="button" onClick={handleClick} className="key zero" value={0}> 0</button>
+              <button
+                type="button"
+                onClick={handleClick}
+                className="key dot"
+                value="."
+              >
+                .
+              </button>
+              <button
+                type="button"
+                onClick={handleClick}
+                className="key one"
+                value={1}
+              >
+                1
+              </button>
+              <button
+                type="button"
+                onClick={handleClick}
+                className="key two"
+                value={2}
+              >
+                2
+              </button>
+              <button
+                type="button"
+                onClick={handleClick}
+                className="key three"
+                value={3}
+              >
+                3
+              </button>
+              <button
+                type="button"
+                onClick={handleClick}
+                className="key four"
+                value={4}
+              >
+                4
+              </button>
+              <button
+                type="button"
+                onClick={handleClick}
+                className="key five"
+                value={5}
+              >
+                5
+              </button>
+              <button
+                type="button"
+                onClick={handleClick}
+                className="key six"
+                value={6}
+              >
+                6
+              </button>
+              <button
+                type="button"
+                onClick={handleClick}
+                className="key seven"
+                value={7}
+              >
+                7
+              </button>
+              <button
+                type="button"
+                onClick={handleClick}
+                className="key eight"
+                value={8}
+              >
+                8
+              </button>
+              <button
+                type="button"
+                onClick={handleClick}
+                className="key nine"
+                value={9}
+              >
+                9
+              </button>
+            </div>
+          </div>
+          <div className="operators">
+            <button type="button" onClick={handleClick} className="key divide" value="÷">/ </button>
+            <button type="button" onClick={handleClick} className="key multiply" value="x">* </button>
+            <button type="button" onClick={handleClick} className="key subtract" value="-">-</button>
+            <button type="button" onClick={handleClick} className="key add" value="+">+ </button>
+            <button type="button" onClick={handleClick} className="key equalTo" value="="> = </button>
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+export default Calculator;
```

Comment:
- [ ] I do not see you using this file anywhere. I think it would be better if you delete it.